packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -64,104 +73,122 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
   )
 
   const PublicationCard = ({ pub }: PublicationCardProps) => (
-    <Grid item xs={12} sm={6} md={4} key={pub.rowId}>
+    <Grid
+      item
+      key={pub.rowId}
+      height={{ xs: 'auto', sm: '275px' }}
+      minWidth={{ xs: '280px', lg: 'initial' }}
+      maxWidth={{ xs: '450px', lg: 'initial' }}
+      sx={{
+        paddingTop: '0px !important',
+        paddingLeft: '0px !important',
+      }}
+    >
       <Box sx={{ height: '100%' }}>
         {isLoading ? (
-          <Skeleton variant="rectangular" height={250} width="100%" />
+          <Skeleton variant="rectangular" height={275} width="100%" />
         ) : (
-          <CardContent>
-            <Box display={'flex'} gap={'6px'} flexWrap={'wrap'}>
-              {pub.values[tagColIndex] &&
-                JSON.parse(pub.values[tagColIndex] as string).map(
-                  (tag: string, index: number) => (
-                    <Typography
-                      key={index}
-                      variant="overline"
-                      sx={{
-                        backgroundColor: 'grey.300',
-                        borderRadius: '3px',
-                        padding: '4px 8px',
-                        border: 'none',
-                        lineHeight: 'initial',
-                      }}
-                    >
-                      {tag}
-                    </Typography>
-                  ),
-                )}
-            </Box>
+          <Box>
```

Comment:
Go ahead and use `div` if you aren't adding any props
```suggestion
          <div>
```